### PR TITLE
check intermediate result

### PR DESCRIPTION
**Pull request by mkovshun**

### CODE REVIEW

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
please provide demo link

**Comment by mkovshun** [comment]
https://mkovshun.github.io/layout_landing-page/

File: src/index.html
Comment (anastasiiavorobiova) [inline src/index.html:1]: Screenshot 2023-09-26 at 16 10 58
Consider fixing validation errors

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added a few comments to fix https://www.loom.com/share/8b207ed6086e4615ab1ea5ed798cd16c

**Comment by mkovshun** [comment]
https://mkovshun.github.io/layout_landing-page/

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. add some hovers to all links
2. make all fields in form required
3. click should start the call
image

**Comment by mkovshun** [comment]
im fix bug with quotes in aside

**Review by plodnikate — APPROVED** [review]
Yep! Super last final result this time! Really good job on it!

**Pull request closed without merging**